stdlib/platform/java/bytecode.wyv

```diff
@@ -0,0 +1,14 @@
+module def bytecode(java: Java)
```

Comment:
Hmm, was this file added by accident?  It seems not related to the IfTSL change.  If it's a pain to remove, I don't mind adding it if it is good code and will part part of your additions eventually.

Reply:
Yep, more of me being bad at git. PR should be fixed now.